doc/RowBasedReplication.md

```diff
@@ -31,27 +31,31 @@ A few binlog events are used:
 The
 [`binlog-row-image` option](https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_row_image) can
 be used to control which rows are used to identify the columns for the Update
-and Delete Rows events. The default setting for that option is to log all columns.
+and Delete Rows events. The default setting for that option is to log all
+columns.
 
 ## Vitess Use of Replication Stream
```

Comment:
s/Replication Stream/MySQL Replication Stream/g ?

Add "MySQL" to the header and in the paragraph below to stress that we're talking about the MySQL replication stream?